File: user-service/src/main/java/rs/edu/raf/dto/EmployeeDTO.java
package rs.edu.raf.dto;

import java.util.List;

public record EmployeeDTO(Long id, String firstName, String lastName, String JMBG, Long dateOfBirth, String gender,
                          String email, String phoneNumber, String address, String position, String department,
                          List<String> roles, Long companyId
)
{ }

File: user-service-kotlin/src/main/kotlin/org/example/userservicekotlin/dto/EmployeeDTO.kt
package org.example.userservicekotlin.dto

data class EmployeeDTO(
    val id: Long?,
    val firstName: String,
    val lastName: String,
    val JMBG: String,
    val dateOfBirth: Long,
    val gender: String,
    val email: String,
    val phoneNumber: String,
    val address: String,
    val position: String,
    val department: String,
    val roles: List<String>,
    val companyId: Long?
)
